src/main/java/org/jabref/gui/push/PushToVScode.java

```diff
@@ -0,0 +1,38 @@
+package org.jabref.gui.push;
+
+import java.nio.file.Path;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.icon.IconTheme;
+import org.jabref.gui.icon.JabRefIcon;
+import org.jabref.gui.preferences.GuiPreferences;
+
+public class PushToVScode extends AbstractPushToApplication {
+
+    public static final String NAME = PushToApplications.VSCODE;
+
+    public PushToVScode(DialogService dialogService, GuiPreferences preferences) {
+        super(dialogService, preferences);
+    }
+
+    @Override
+    public String getDisplayName() {
+        return NAME;
+    }
+
+    @Override
+    public JabRefIcon getApplicationIcon() {
+        return IconTheme.JabRefIcons.APPLICATION_VSCODE;
+    }
+
+    @Override
+    protected String[] getCommandLine(String keyString) {
+        // No idea to insert the citation to the open tex file in vscode
```

Comment:
Refs #6775

Reply:
Thanks

Reply:
As a seperate PR. 

But link the issue in the TODO.